utils/mock.js

```diff
@@ -45,6 +54,13 @@ function mockQuery (schema, mockedSchema, parsedQuery, fixture, saveFixture, glo
       return mockedQuery
```

Comment:
should not be the response shape `{data, errors}` ? (for example for a partial response).
```js
const data = { [queryName]: mock };
...
return {data, errors};
```